fractal_server/app/models/job.py

```diff
@@ -79,6 +81,9 @@ class ApplyWorkflow(_ApplyWorkflowBase, SQLModel, table=True):
             (Mapper attribute)
     """
 
+    class Config:
```

Comment:
I'm always confused by this. Is it because of `Column(JSON)`?

Reply:
exactly, not a standard type for sqlalchemy https://docs.sqlalchemy.org/en/20/core/type_basics.html#generic-camelcase-types

Reply:
Thanks!